lib/info.js

```diff
@@ -179,8 +179,7 @@ const gotConfig = (id, options, additional, config, fromEmbed, callback) => {
       info.fmt_list.map((format) => format.split('/')) : [];
 
     info.formats = parseFormats(info);
-
-    // Add additional properties to info.
+      // Add additional properties to info.
```

Comment:
this is still indented from the previous commit. please unindent

Reply:
got it, thanks